modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/Qt5CPPGenerator.java

```diff
@@ -337,6 +337,11 @@ public String toModelName(String type) {
     }
 
     @Override
+    public String toParamName(String name) {
+        return name.replaceAll("[^A-Za-z0-9_]", "_").replaceAll("_+", "_");
```

Comment:
One suggestion: What about reusing similar logic in Ruby toParamName/toVarName to remove illegal characters and convert the name into snake_case? https://github.com/swagger-api/swagger-codegen/blob/1dd9ee39af6c4e2f9c016a94d16ca47a88a9cb0d/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/RubyClientCodegen.java#L434-L459